logger/logger.go

```diff
@@ -217,7 +217,14 @@ func (l *Logger) logWithLevel(logLvl LogLevel, data map[string]interface{}) {
 
 // New creates a *logger.Logger. Default values are Debug LogLevel, kayvee Formatter, and std.err output.
 func New(source string) *Logger {
-	logObj := Logger{}
+	return NewWithContext(source, nil)
+}
+
+// NewWithContext creates a *logger.Logger. Default values are Debug LogLevel, kayvee Formatter, and std.err output.
+func NewWithContext(source string, context M) *Logger {
+	logObj := Logger{
+		globals: context,
```

Comment:
updated in https://github.com/Clever/kayvee-go/pull/22/commits/16efbb333cb7bec355ea982ca94b8470825360e4